react-elements/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "react-elements",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "webpack": "^5.56.1",
+    "webpack-cli": "^4.8.0"
```

Comment:
`webpack` and `webpack-cli` should be installed as `devDependencies`. You can move them by running the following command: `npm install --save-dev package-name`, replace `package-name` with the actual packages name.